### ISSUE TITLE
SQL not supported

### ISSUE THREAD

**Issue by suntong** [micro-editor/comment-plugin#25]
SQL not supported yet. 

SQL are commented as "`# `" instead of "`-- `".

**Comment by bharatagarwal** [comment]
Add `ft["sql"] = "-- %s" in `comment.lua` and you'll be good to go.

**Comment by suntong** [comment]
Yeah I know, I've even added mine: https://github.com/micro-editor/comment-plugin/pull/26
then close it after seeing #23.

**Comment by bharatagarwal** [comment]
Oops, my bad, didn't realise you were the author of the pull request.

**Comment by bharatagarwal** [comment]
Oh, alright, I didn't notice the closed pull request.
